src/po/Submakefile

```diff
@@ -55,13 +55,14 @@ TCLSRCS := \
 	../tcl/tooledit.tcl
 
 TOOLI18NSRCS := \
-    ../lib/python/gladevcp/iconview.py \
-    ../lib/python/gladevcp/hal_mdihistory.py \
+	../lib/python/gladevcp/iconview.py \
```

Comment:
Is this a tab vrs space change?
I understand adding calculator.glade but don't understand the other changes.

Reply:
>Is this a tab vrs space change?

Exactly! Only space changes except adding calculator.glade. Which other changes do you not understand?
I just recognized that I accidentally commited some files (gmoccapy.pot, linuxcnc.pot, sim.tbl) - maybe these ones?
I already corrected this.